docs/components/connectors/use-connectors/index.md

```diff
@@ -218,6 +191,35 @@ In that case, you could declare `Result Expression` as follows:
 }
 ```
 
+## Activation
```

Comment:
Only changed the order of the sections here as the example above is now where it belongs to